starknet_devnet/starknet_wrapper.py

```diff
@@ -450,7 +481,11 @@ async def invoke(self, external_tx: InvokeFunction):
     async def call(self, transaction: CallFunction):
```

Comment:
Is `call` really the only place that needs to be modified to use `mempool_state`?

Reply:
Let's discuss it later on a call since the user can interact with contract only when `deploy` is called and it's accepted on l2.